docs/manual/src/docs/asciidoc/_includes/servlet/preface/java-configuration.adoc

```diff
@@ -819,56 +817,6 @@ JwtDecoder jwtDecoder() {
 ```
 This will keep all the defaults, except it will override the default claim converter for `sub`.
 
-[[oauth2resourceserver-claimsetmapping-add]]
```

Comment:
It doesn't seem to me that this PR is about changing how `MappedClaimSetConverter` works. We should defer changing its documentation to maybe #6800.